2005/chuuuul/ArchitectureStudy/app/src/main/java/com/example/architecture/util/ConstValue.kt

```diff
@@ -6,6 +6,9 @@ class ConstValue {
         const val NO_IMAGE_URL = "https://ssl.pstatic.net/static/movie/2012/06/dft_img133x190.png"
 
         const val MOVIE_SHARED_PREFERENCE_NAME = "MoviesSharedPreferenceName"
+
+        const val AUTO_SEARCH_TIME: Long = 3000
```

Comment:
```suggestion
        const val AUTO_SEARCH_TIME = 3000L
```
통일성을 위해 위와 같이 쓰면 어떨까요?

Reply:
3689ff9
감사합니다. 수정했습니다.